mlflow/environment_variables.py

```diff
@@ -100,55 +103,65 @@ def get(self):
 
 #: Specifies the timeout in seconds for MLflow HTTP requests
 #: (default: ``120``)
-MLFLOW_HTTP_REQUEST_TIMEOUT = _EnvironmentVariable("MLFLOW_HTTP_REQUEST_TIMEOUT", int, 120)
+MLFLOW_HTTP_REQUEST_TIMEOUT = _EnvironmentVariable(
+    "MLFLOW_HTTP_REQUEST_TIMEOUT", int, 120)
 
 #: Specifies whether MLFlow HTTP requests should be signed using AWS signature V4. It will overwrite
 #: (default: ``False``). When set, it will overwrite the "Authorization" HTTP header.
 #: See https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html for more information.
-MLFLOW_TRACKING_AWS_SIGV4 = _BooleanEnvironmentVariable("MLFLOW_TRACKING_AWS_SIGV4", False)
+MLFLOW_TRACKING_AWS_SIGV4 = _BooleanEnvironmentVariable(
+    "MLFLOW_TRACKING_AWS_SIGV4", False)
 
 #: Specifies the chunk size to use when downloading a file from GCS
 #: (default: ``None``). If None, the chunk size is automatically determined by the
 #: ``google-cloud-storage`` package.
-MLFLOW_GCS_DOWNLOAD_CHUNK_SIZE = _EnvironmentVariable("MLFLOW_GCS_DOWNLOAD_CHUNK_SIZE", int, None)
+MLFLOW_GCS_DOWNLOAD_CHUNK_SIZE = _EnvironmentVariable(
+    "MLFLOW_GCS_DOWNLOAD_CHUNK_SIZE", int, None)
 
 #: Specifies the chunk size to use when uploading a file to GCS.
 #: (default: ``None``). If None, the chunk size is automatically determined by the
 #: ``google-cloud-storage`` package.
-MLFLOW_GCS_UPLOAD_CHUNK_SIZE = _EnvironmentVariable("MLFLOW_GCS_UPLOAD_CHUNK_SIZE", int, None)
+MLFLOW_GCS_UPLOAD_CHUNK_SIZE = _EnvironmentVariable(
+    "MLFLOW_GCS_UPLOAD_CHUNK_SIZE", int, None)
```

Comment:
Can you avoid unrelated changes?

Reply:
I am trying to pass the DCO check. I am stuck in rebasing.